src/main/webapp/data-visualization.js

```diff
@@ -23,6 +23,20 @@ async function getGeoData(locationInfo, isCountyQuery) {
   }
 }
 
+// Create dropdown menu of couties in queried state
+// for yearly population change in county chart.
+function createCountyDropdown(geoDataFeatures) {
+  const datalist = document.getElementById('county');
+  geoDataFeatures = geoDataFeatures.reverse(); // Make dropdown alphabetical
+  datalist.innerHTML = '';
+  for (let i = 0; i < geoDataFeatures.length; i++) {
+    const optionElem = document.createElement('option');
+    optionElem.value = geoDataFeatures[i].properties.name;
+    optionElem.setAttribute('data-value', geoDataFeatures[i].id.slice(2, 5));
```

Comment:
why (2,5)?

Reply:
Added a comment about this, it's to get last 3 digits of county id which is county number without the state number